src/networking.c

```diff
@@ -264,72 +209,60 @@ cc_oci_netlink_run(const gchar *cmd_line) {
  * \return \c true on success, else \c false.
  */
 gboolean
-cc_oci_network_create(const struct cc_oci_config *const config) {
+cc_oci_network_create(const struct cc_oci_config *const config,
+		      struct netlink_handle *const hndl) {
 	struct cc_oci_net_if_cfg *if_cfg = NULL;
-	gchar cmd_line[1024];
 	guint index = 0;
 
 	if (config == NULL) {
 		return false;
 	}
 
 	for (index=0; index<g_slist_length(config->net.interfaces); index++) {
+		guchar mac[6] = {0x02, 0x00, 0xCA, 0xFE, 0x00, (guchar)index};
```

Comment:
I guess this hard-coded MAC address is safe since the container has its own network ns. If so, could you add a comment explaining this?